ui/css/logviewer.css

```diff
@@ -98,6 +98,10 @@ body {
     background: #fbe3e3;
 }
 
+.lv-log-container > div.text-selected {
+    background: yellow;
```

Comment:
Let's match Github here, `#f8eec7` and see how that looks. Keep in mind it will need to apply successfully over both regular lines (our alternating white/grey) and also over selected error lines (pink), per the example lines below.

![lines](https://cloud.githubusercontent.com/assets/3660661/9747240/9c261d84-564a-11e5-9778-367bfef288cf.jpg)